### DISCUSSION TITLE
Adverse Event

### DISCUSSION THREAD

**Discussion by WarrenArndt** [hl7au/au-fhir-core#72] [AU Core R1 Resource Priority List - CLOSED]
AU equivalent of Adverse Event in FHIR
https://build.fhir.org/adverseevent.html

**Comment by GaryPascoe** [comment]
Agree with @WarrenArndt - and priority and scope should reflect and support the Australian government's mandate for improving and increasing information sharing about medical device safety and effectiveness which will see mandatory reporting of medical device adverse events by Australian hospitals and healthcare facilities.

The legislation was passed March 2023, making it mandatory for Australian public and private hospitals and any other health facilities (prescribed by the regulations) to report medical devices related adverse events to the TGA.  

While the Regulations to support this implementation are due by 22 March 2025, it will have far reaching impact across the healthcare sector and healthcare facilities will need considerable time to prepare relevant IT systems and data exchange, which supports early inclusion in FHIR Core.

The TGA and Australian Commission on Safety and Quality in Health Care are responsible for the implementation - see here for more [Progress update: Mandatory reporting of adverse events by healthcare facilities](https://www.tga.gov.au/resources/publication/publications/progress-update-mandatory-reporting-adverse-events-healthcare-facilities#:~:text=On%2021%20March%202023%2C%20the%20legislation%20passed%20making,support%20implementation%20is%20due%20by%2022%20March%202025.)
